runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -2175,13 +2182,83 @@ private static String longBossName(String boss)
 			// Tombs of Amascut
 			case "toa":
 				return "Tombs of Amascut";
+			case "toa 1":
+			case "toa solo":
+				return "Tombs of Amascut Solo";
+			case "toa 2":
+			case "toa duo":
+				return "Tombs of Amascut 2 players";
+			case "toa 3":
+				return "Tombs of Amascut 3 players";
+			case "toa 4":
+				return "Tombs of Amascut 4 players";
+			case "toa 5":
+				return "Tombs of Amascut 5 players";
+			case "toa 6":
+				return "Tombs of Amascut 6 players";
+			case "toa 7":
+				return "Tombs of Amascut 7 players";
+			case "toa 8":
+				return "Tombs of Amascut 8 players";
 			case "toa entry":
 			case "toa entry mode":
 				return "Tombs of Amascut Entry Mode";
+			case "toa entry 1":
+			case "toa entry solo":
+			case "Tombs Of Amascut - Entry Solo":
+				return "Tombs of Amascut Entry Solo";
```

Comment:
I don't think any of this is right except for 8, because they are missing the word `Mode`? You should have a test for at least one of these.